Automate Your Page/css/style.css

```diff
@@ -0,0 +1,44 @@
+/* In all HTMl file */
+* {
+	font-family: Helvetica;
+	-webkit-box-sizing: border-box;
+	-moz-box-sizing: border-box;
+	-ms-box-sizing: border-box;
+	box-sizing: border-box;
+}
+
+/* Headerpage in the HTMl file */
+#pageheader {

```

Comment:
In CSS, there is no reason to use an id selector over a class selector. CSS class selectors are better in every way, even if you only use it once.